frontend/src/app/AppRoutes.tsx

```diff
@@ -115,7 +111,6 @@ const AppRoutes: React.FC = () => {
         <Route path="/modelRegistry/*" element={<ModelRegistryRoutes />} />
 
         <Route path={globPipelinesAll} element={<GlobalPipelinesRoutes />} />
-        <Route path={globPipelineRunsAll} element={<GlobalPipelineRunsRoutes />} />
```

Comment:
with this route gone we need to remove the navbar for it

Reply:
I checked the code and it's already gone.